server/src/test/java/org/apache/druid/metadata/TestDerbyConnector.java

```diff
@@ -135,5 +136,23 @@ public Supplier<MetadataStorageTablesConfig> metadataTablesConfigSupplier()
     {
       return dbTables;
     }
+
+    public Integer updateSegmentsTable(String sqlFormat, Object... args)
```

Comment:
Does the return value have to be a boxed integer?

Reply:
The javadoc of this method should mention that the `sqlFormat` should have a `%s` placeholder for the table name and a `?` for each of the query args.

Reply:
Added javadocs and removed the boxed type